src/store/modules/job/getters.ts

```diff
@@ -57,6 +57,18 @@ const getters: GetterTree <JobState, RootState> = {
     isMiscellaneousJobsScrollable: (state) => {
       return state.miscellaneous.list?.length > 0 && state.miscellaneous.list?.length < state.miscellaneous.total
     },
+    getMoreJobs (state){
+      const draftOnlyJobs = state.more.draft.list.reduce((jobs: any, draftJob: any) => {
+        state.more.pending.list.find((pendingJob: any) => {
+          if(draftJob.systemJobEnumId !== pendingJob.systemJobEnumId ) {
+            jobs.push(draftJob);
```

Comment:
This logic will display both pending and draft jobs in the more jobs section, but we need to display pending job if exist, otherwise draft job.

Reply:
Ok sir